sdk/testing/stepwise/legacy.go

```diff
@@ -220,7 +229,10 @@ func Test(tt TestT, c TestCase) {
 
 	// Create an HTTP API server and client
 	ln, addr := http.TestServer(nil, core)
-	defer ln.Close()
+	defer func() {
+		// intentionally ingore this error
+		_ = ln.Close()
```

Comment:
Calling `defer ln.Close()` should also ignore the error.

Reply:
Changed in 14557d1bf